_workinggroups/visualization.md

```diff
@@ -2,11 +2,12 @@
 title: "Visualization"
 author: "Riccardo Maria Bianchi"
 layout: default
+redirect_from: /activities/training.html
 ---
 
 # Visualization
 
-The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh*). 
+The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh*).
```

Comment:
developers

Reply:
Ha - not my mistake, but thanks for spotting it (my editor likes to strip trailing whitespace, sometimes that's a good thing!).